render/SongRender.ts

```diff
@@ -88,15 +88,35 @@ export class SongRender {
                       song.sourceName = result.sourceName;
                     }
                     this.displaySong(song);
-                    this['$nextTick'](() => {
-                        document.getElementById("lyricBox").scrollTop = 0;
-                    });
+                    console.log(this.settings.autoScroll);
+                    if(this.settings.autoScroll) {
+                        this['$nextTick'](() => {
+                            document.getElementById("lyricBox").scrollTop = 0;
```

Comment:
could you move this logic to a new function? and only do the work if the setting is enabled.